lib/worker.go

```diff
@@ -406,6 +402,23 @@ func (w *DefaultBaseWorker) initMessageHandlers(ctx context.Context) error {
 		log.L().Panic("duplicate handler",
 			zap.String("topic", topic))
 	}
+
+	topic = WorkerStatusChangeRequestTopic(w.masterID, w.id)
+	ok, err = w.messageHandlerManager.RegisterHandler(
+		ctx,
+		topic,
+		&StatusChangeRequest{},
+		func(sender p2p.NodeID, value p2p.MessageValue) error {
+			msg := value.(*StatusChangeRequest)
+			return w.Impl.OnMasterMessage(topic, msg)
```

Comment:
if the onMasterMessage costs a long time, the message server won't work

Reply:
Good catch, I will add an asynchronous mechanism